src/checkoutservice/main.go

```diff
@@ -184,6 +206,17 @@ func (cs *checkoutService) PlaceOrder(ctx context.Context, req *pb.PlaceOrderReq
 		Items:              prep.orderItems,
 	}
 
+	shippingCostFloat, _ := strconv.ParseFloat(fmt.Sprintf("%d.%02d", prep.shippingCostLocalized.GetUnits(), prep.shippingCostLocalized.GetNanos()/1000000000), 64)
+	totalPriceFloat, _ := strconv.ParseFloat(fmt.Sprintf("%d.%02d", total.GetUnits(), total.GetNanos()/1000000000), 64)
+
+	span.SetAttributes(
+		attribute.String("app.order.id", orderID.String()),
+		shippingTrackingAttribute,
+		attribute.Float64("app.order.shipping.cost", shippingCostFloat),
```

Comment:
In the emailservice we have `app.shipping.cost.total`.
We could agree in one and apply in both services.

Reply:
I think the branch will need to be updated before merging so leaving this for now. @puckpuck when you have a chance take a look. @julianocosta89 this could be another use of your centralized attribute table. Ensure consistency across